services/web/server/src/simcore_service_webserver/products.py

```diff
@@ -0,0 +1,59 @@
+import logging
+import re
+from typing import Optional
+
+from aiohttp import web
+
+from servicelib.application_setup import ModuleCategory, app_module_setup
+
+from .__version__ import api_vtag
+from .constants import RQ_PRODUCT_KEY
+
+log = logging.getLogger(__name__)
+
+# TODO: <--- This is defined by services/web/client/compile.json
+FE_APPS = ["osparc", "s4l", "tis", "explorer", "apiviewer", "testtapper"]
```

Comment:
"explorer", "apiviewer", "testtapper" were removed

Reply:
so I guess these will go in the database instead of hard-coded..

Reply:
i know, it is updatedn and there is also a test that makes sure that the correspnoding front-end apps exist